modules/imaging_qc/jsx/imagingQCIndex.js

```diff
@@ -217,7 +217,10 @@ class ImagingQCIndex extends Component {
     } else {
       return (
         <div>
-          <h3>An error occurred while loading the page.</h3>
+          <h3>
+            The MRI parameter form instrument must be installed
```

Comment:
What if an error other than the MRI parameter_form not being setup occurs?

Reply:
Currently any error from the php script leads to this error being shown. I am unable to figure out how to make a connection between php file and the jsx code in-order to tell it what the exact error is. Any help on this would be really appreciated :)

Reply:
You would need to catch the exception in PHP, and then return a JSON error response. The JSX could then display that error message with the "An error occurred" as a fallback.